samples/meetings-share-to-stage-signing/csharp/Source/MeetingSigning.Infrastructure/Data/Repositories/UserRepository.cs

```diff
@@ -66,9 +55,9 @@ public async Task<User> GetUser(string userId)
         }
 
         /// <inheritdoc/>
-        public async Task<bool> UserExists(string userId)
+        public async Task<bool> UserExists(string id)
```

Comment:
I'm probably going to ignore this here, mainly because it is just a sample, and it's such an edge case. I did add a comment in samples/meetings-share-to-stage-signing/csharp/Source/MeetingSigning.Domain/Models/User.cs that touches on it